Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs

```diff
@@ -269,7 +347,7 @@ private void UnpackRegularFile(Pack1Meta.FileEntry file, Action<double> onProgre
 
             DecompressorCreator decompressorCreator = ResolveDecompressor(_metaData);
 
-            using (var fs = new FileStream(_packagePath, FileMode.Open))
+            using (var fs = new FileStream(_packagePath, FileMode.Open, FileAccess.Read, FileShare.Read))
```

Comment:
Is this change related to the thread pool support?

Reply:
yes